scripts/sendpayment.py

```diff
@@ -172,17 +172,19 @@ def main():
     # the sync call here will now be a no-op:
     wallet_service.startService()
 
-    # Dynamically estimate a realistic fee.
+    # Dynamically estimate a realistic fee, for coinjoins.
     # At this point we do not know even the number of our own inputs, so
     # we guess conservatively with 2 inputs and 2 outputs each.
-    fee_per_cp_guess = estimate_tx_fee(2, 2, txtype=wallet_service.get_txtype())
-    log.debug("Estimated miner/tx fee for each cj participant: " + str(
-        fee_per_cp_guess))
+    if options.makercount != 0:
+        fee_per_cp_guess = estimate_tx_fee(2, 2,
+                                           txtype=wallet_service.get_txtype())
+        log.debug("Estimated miner/tx fee for each cj participant: " + str(
```

Comment:
```suggestion
        log.debug("Estimated miner/tx fee for each cj participant: " + btc.amount_to_str(
```

Reply:
Agreed, done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/750/commits/8cdf3e0a4ac218327fb9928d81273b2ca311fc12